api/api.py

```diff
@@ -499,3 +501,30 @@ def bids(user, key):
             "Your bid will be confirmed once you scan the QR code.",
         ] + (["Started following the auction."] if started_following else []),
     })
+
+
+@api_blueprint.route('/api/stores/<string:twitter_username>/auctions', methods=['GET'])
+def store_auctions_view(twitter_username):
+    user = m.User.query.filter_by(twitter_username=twitter_username).first()
+    if user:
+        return jsonify({'auctions': [a.to_dict() for a in sorted(user.auctions.all(), key=lambda a: len(a.bids), reverse=True)]})
+
+
+@api_blueprint.route('/api/stores/<string:twitter_username>', methods=['GET'])
+def store_view(twitter_username):
```

Comment:
This endpoint is misnamed. It is called /api/stores..., but returns a *user*. I think we should rename it. to /api/users...